### PR TITLE
Add Phpactor as PHP's Language Server

### PR DESCRIPTION
**Pull request by zonuexe**
[Phpactor](https://github.com/phpactor/phpactor) is an implementation of PHP's Language Server.

### CODE REVIEW

**Comment by manateelazycat** [comment]
Can you add lsp-bridge-php-lsp-server like lsp-bridge-python-lsp-server does, user can customize lsp-bridge-php-lsp-server to choose PHP LSP server.

File: lsp-bridge.el [hunk 1]
```diff
@@ -323,6 +323,11 @@ Then LSP-Bridge will start by gdb, please send new issue with `*lsp-bridge*' buf
   "Default LSP server for C language, you can choose `clangd' or `ccls'."
   :type 'string)
 
+(defcustom lsp-bridge-php-lsp-server "intelephense"
+  "Default LSP server for PHP language, you can choose `intelephense' or `phpactor'."
+  :type 'string
+  :safe #'stringp)
```
Comment (zonuexe) [inline lsp-bridge.el:329]: I want to switch servers in `.dir-locals.el` per project in PHP, so I mark it as a [safe variable](https://www.gnu.org/software/emacs/manual/html_node/emacs/Safe-File-Variables.html) by setting `:safe`.

**Comment by zonuexe** [comment]
@manateelazycat Thank you for reviewing. I added custom variable and rebased and squashed this branch.

**Comment by manateelazycat** [comment]
Awesome!

**Pull request merged**